holopy/scattering/theory/dda.py

```diff
@@ -36,11 +33,14 @@
 import time
 import warnings
 
-from holopy.scattering.theory.scatteringtheory import ScatteringTheory
-from holopy.scattering.scatterer import (Ellipsoid, Capsule, Cylinder,
-                                        Bisphere, Sphere, Scatterer, Spheroid)
+import numpy as np
+
 from holopy.core.utils import ensure_array
-from holopy.core.errors import DependencyMissing
+from holopy.scattering.scatterer import (
+    Ellipsoid, Capsule, Cylinder, Bisphere, Sphere, Scatterer, Spheroid)
+from holopy.scattering.errors import DependencyMissing
```

Comment:
This should be switched back to importing from `holopy.core.errors` since I moved it when rejigging dependencies. This is causing unittests to fail on Travis.

Reply:
Oops, my bad :) I had un-commited changes on my machine. Should be fixed with commit f659172